platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/ConfigurationCacheUnsupportedTypesIntegrationTest.groovy

```diff
@@ -259,7 +261,7 @@ class ConfigurationCacheUnsupportedTypesIntegrationTest extends AbstractConfigur
         ScriptClassLoader                     | ClassLoader                    | "getClass().classLoader"
         Thread                                | Thread                         | "Thread.currentThread()"
         DefaultThreadFactory                  | ThreadFactory                  | "java.util.concurrent.Executors.defaultThreadFactory()"
-        FinalizableDelegatedExecutorService   | Executor                       | "java.util.concurrent.Executors.newSingleThreadExecutor().tap { shutdown() }"
+        executorServiceTypeOnCurrentJvm()     | Executor                       | "java.util.concurrent.Executors.newSingleThreadExecutor().tap { shutdown() }"
```

Comment:
Is the import above broken on 21?

Reply:
Technically yes, but imports are gone by the point it's executed. I'll remove it in case we ever start compiling on 21.